src/ServicePulse.Host/app/js/views/failed_groups/failed-groups-view.html

```diff
@@ -29,7 +29,8 @@ <h6>
                                 <div class="col-sm-12 no-side-padding">
                                     <div class="row box-header">
                                         <div class="col-sm-12 no-side-padding">
-                                            <p class="lead break"> {{group.originator}}</p>
```

Comment:
What is written below is correct. But a shorter way of doing that would be:
<p class="lead break"> {{group.originator || 'Selection of individual message(s)'}}</p>